docs/language/informal/enhanced-type-promotion.md

```diff
@@ -125,7 +125,12 @@ The language specifies type promotion out of three pieces:
     closure that is passed to it. To avoid cases like this, the spec defines
     certain variables to be off limits for type promotion.
 
-This proposal keeps the same three general pieces, but refines each of them.
+4.  **Which pairs of types -- declared and promoted -- are allowed for
+    promotion.** The spec says that promotion only applies if the promoted type
+    is a subtype of the declared type.
+
```

Comment:
The spec says more specific, not subtype. This is why you cannot promote `Iterable<String>` to `List`.  In simple cases, like this one, there is only one way for a `List` to be an `Iterable<String>`, namely a `List<String>` thus promoting to `List<String>` would avoid the need for an intersection type (`Iterable<String>&List`) in this case.